### PR TITLE
New zingolib version post mob-prelease-1.3.1

### PR DESCRIPTION
**Pull request by juanky201271**
I sent an APK to @edicksonjga ...

### CODE REVIEW

File: rust/lib/src/lib.rs [hunk 1]
```diff
@@ -24,19 +24,7 @@ fn lock_client_return_seed(lightclient: LightClient) -> String {
...
-        .as_ref()
-        .unwrap()
-        .do_seed_phrase_sync()
-    {
-        Ok(s) => s.dump(),
-        Err(e) => {
-            return format!("Error: {}", e);
-        }
-    };
+    let seed = execute("seed".to_string(), "".to_string());
```
Comment (Oscar-Pepper) [inline rust/lib/src/lib.rs]: in rust you can actually return this directly if you delete the closure ( ; )
i.e.
let seed = execute("seed".to_string(), "".to_string());
seed

is the same as:
execute("seed".to_string(), "".to_string())

Comment (juanky201271) [reply rust/lib/src/lib.rs]: Good point... and thanks. I changed.

**Review by zancas — DISMISSED** [review]

File: android/app/src/androidTest/java/org/ZingoLabs/Zingo/RustFFITest.kt [hunk 2]
```diff
@@ -18,7 +18,8 @@ object Ufvk {
 
 data class InitFromSeed (
     val seed : String,
-    val birthday : Long
+    val birthday : Long,
+    val account_index: Long
```
Comment (zancas) [inline android/app/src/androidTest/java/org/ZingoLabs/Zingo/RustFFITest.kt:22]: This has to do with the addition of greater than initial index addresses being supported for a given seed.  The idea then is that more addresses are supported from a single seed.  Each address has a unique index.

**Review by Oscar-Pepper — DISMISSED** [review]

**Comment by juanky201271** [comment]
- the `version` format in zingolib changed, from `mob-release` to `mob-prerelease` -> I needed to change the test.
- the `seed` result in zingolib changed:
-- from: "Error: xxxxx" or "{seed: ''", birthday: ""}"
-- to: "{error: ""}" or "{seed: ''", birthday: "", account_index: ""}"
- I needed to change zingo-mobile when is using a UFVK, the warning of `watch-only` changed as well.

**Review by Oscar-Pepper — APPROVED** [review]
awesome!

**Pull request merged**